lib/onc_certification_g10_test_kit.rb

```diff
@@ -140,6 +141,9 @@ def self.well_known_route_handler
 
     group from: 'multi_patient_api'
 
+    group from: 'multi_patient_api_stu2',
```

Comment:
The SVAP stuff is all in, as are the feature flags, so this should be updated to use them.